Backpack/BottomSheet/Classes/BottomSheet.swift

```diff
@@ -213,7 +217,7 @@ public final class BPKBottomSheet: NSObject {
     /// It can be useful, for example, when changing the inner constraints of the `contentViewController`
     /// and bottom sheet needs to be resized to fit the content.
     public func updateLayout() {
-        floatingPanelController.updateLayout()
+        floatingPanelController.invalidateLayout()
     }
```

Comment:
Is this even needed? shouldn't constraints just work without this?

Reply:
If we don't do this, then the 'resize' example stops working